lib/ui/boxen.ts

```diff
@@ -0,0 +1,39 @@
+import { Options } from 'boxen';
+import { uiLogger } from './logger';
+import { UI_COLORS } from './index';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { lib } from '../../lang/en';
+
+const defaultOptions: Options = {
+  titleAlignment: 'left',
+  borderColor: UI_COLORS.MARIGOLD,
+  margin: 1,
+  padding: 1,
+  textAlignment: 'left',
+  borderStyle: 'round',
+};
+
+export async function logInBox({
+  contents,
+  options,
+  fallBackToNoBox = false,
```

Comment:
I love this variable name lol, but I'd also be fine with removing it and always falling back to no box. Or defaulting to true. We should probably be falling back to no box in the fire alarm middleware too.

Reply:
Haha, I had such a hard time naming it.  I'll make it fallback by default